### DISCUSSION TITLE
Conky for mac

### DISCUSSION THREAD

**Discussion by simonmcnair** [brndnmtthws/conky#1322] [Q&A] [answered]
Hi,
I'm new to Conky, and new to mac.  I have managed to compile and install conky.  I was just wondering if there is any help to have conky auto start and/or any themes that you're aware of.

I tried conky-for-macos but although it did a good job of detailing the compile requirements I couldn't use any of the images/apps as it is unmaintained.  Also I couldn't see any themes.

I would appreciate any input.
Cheers

**Comment by Caellian** [comment, marked as answer]
Sorry for late response.

- [Autostart commands on startup](https://superuser.com/questions/229773/run-command-on-startup-login-mac-os-x)

As for themes, most of them _should_ work on MacOS for the most part. There's a lot of linux specific variables mostly because majority of users use linux, but apart from that, most themes should just work. You'll get an error message if you try to use a variable that isn't available.
